charts/rancher-backup-crd/rancher-backup-crd/2.0.0+up2.0.0-rc1/Chart.yaml

```diff
@@ -8,4 +8,4 @@ appVersion: 2.0.0-rc1
 description: Installs the CRDs for rancher-backup.
 name: rancher-backup-crd
 type: application
-version: 2.0.0-rc1
+version: 2.0.0+up2.0.0-rc1
```

Comment:
Confused why all the versions that are in line are still getting the `up` part. Will that go away once its not an RC anymore ?

Reply:
Now the `+up` part is added because 2.0.0 != 2.0.0-rc1
and it will go away when we use the released version (rancher/rancher-backup 2.0.0 in this case).